amundsen_application/static/js/components/TableDetail/DataPreviewButton/styles.scss

```diff
@@ -7,8 +7,9 @@
     min-height: 90%;
 
     .modal-title {
-      font-family: $font-family-sans-serif-bold;
-      font-weight: $font-weight-sans-serif-bold;
+      font-size: 20px;
+      font-family: $font-family-header;
+      font-weight: $font-weight-header-bold;
```

Comment:
Seems like the `<Modal.Title>` component automatically renders as an `h4` so I am manually defining what would otherwise be an `h3`.